tests/filtertransformers/test_mongo.py

```diff
@@ -221,6 +221,8 @@ def test_list_properties(self):
         # Comparisons of list properties
         self.assertEqual(self.transform("list HAS < 3"), {})
         self.assertEqual(self.transform("list HAS ALL < 3, > 3"), {})
+        self.assertEqual(self.transform("list HAS ANY > 3, < 6"), {})
+        self.assertEqual(self.transform("list LENGTH 3"), {})
```

Comment:
Shouldn't this one work now? This whole unittest is currently being skipped, but perhaps its better to unskip it and check that all the things that aren't currently implemeneted do indeed raise `NotImplementedError`s?

https://github.com/Materials-Consortia/optimade-python-tools/blob/9ef5b68c961c35959dbcca24c6a5f53f972efc16/tests/filtertransformers/test_mongo.py#L219

Reply:
I like this. I will try to do this, unless you beat me to it :sweat_smile:

Reply:
It's all yours!